aws/alarms/athena.tf

```diff
@@ -134,13 +134,19 @@ resource "aws_iam_role_policy" "athena_dynamodb_policy" {
       {
         "Action" : [
           "dynamodb:DescribeTable",
-          "dynamodb:ListSchemas",
```

Comment:
This one just needs to move up in the Glue statement above and become `glue:ListSchemas`